src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -41,7 +41,7 @@ def test_method_get_self_cal_supported(session):
 
 def test_get_self_cal_last_date_and_time():
     try:
-        with nifgen.Session('', False, 'Simulate=1, DriverSetup=Model:5421;BoardType:PXI') as session:
+        with nifgen.Session('', False, 'Simulate=1, DriverSetup=Model:5421;BoardType:PXI') as session:  # Simulated 5433 returns unrecoverable error when calling get_self_cal_last_date_and_time()
```

Comment:
this comment doesn't match the comment on line 48

Reply:
Error codes are different for 5413/23/33 and other FGen
Only on 5413/23/33 calling some methods get_self_cal_last_date_and_time, get_ext_cal_last_date_and_time returns unrecoverable failure while other FGen devices returns operations not supported in simulated devices.

Reply:
That's ok, but the comment still doesn't match the comment on line 48.